edu_edfi_airflow/dags/edfi_resource_dag.py

```diff
@@ -521,6 +528,7 @@ def build_default_edfi_to_snowflake_task_group(self,
                     get_key_changes=get_key_changes,
                     min_change_version=self.xcom_pull_template_get_key(get_cv_operator, endpoint) if get_cv_operator else None,
                     max_change_version=airflow_util.xcom_pull_template(self.newest_edfi_cv_task_id),
+                    reverse_paging=self.get_deletes_cv_with_deltas,
```

Comment:
I want to be careful here... we still want to use reverse paging for resources, even if the deletes can't be reverse paged.
```
reverse_paging=self.get_deletes_cv_with_deltas if get_deletes else True
```

Reply:
thanks! fixed b699830581df5241c53398a858ea9cde22be1641